include/aws/common/private/cpu_usage_sampler_private.h

```diff
@@ -12,16 +12,16 @@
 /**
  * The VTable for the CPU sampler in cpu_usage_sampler.h
  */
-struct aws_cpu_sampler_vtable {
-    int (*aws_get_cpu_sample_fn)(struct aws_cpu_sampler *sampler, double *output);
-    void (*aws_cpu_sampler_destroy)(struct aws_cpu_sampler *sampler);
+struct aws_system_cpu_sampler_vtable {
+    int (*aws_get_cpu_sample_fn)(struct aws_system_cpu_sampler *sampler, double *output);
+    void (*aws_system_cpu_sampler_destroy)(struct aws_system_cpu_sampler *sampler);
```

Comment:
trivial: just name these vtable members like:
- ~aws_get_cpu_sample_fn~ -> `get_sample`
- ~aws_system_cpu_sampler_destroy~ -> `destroy`

If we're `typedef`ing a callback we do the full aws_ name and put _fn at the end. But for vtable member names there's no need for all the fanfare